src/Support/SelectFields.php

```diff
@@ -147,6 +148,11 @@ protected static function handleFields(
                 continue;
             }
 
+            if ($field === self::MORPH_TYPE_KEY) {
```

Comment:
How about merging with the if condition before, as in `if ($field === … || $field === …)`. Seems to be both actions are the same, anyway?

Reply:
Which begs the question: do we need the separate constant `MORPH_TYPE_KEY` really? It behaves exactly the same as the other constant.

Mayne we should change the constant to better express the intent, something like: `ALWAYS_RELATION_KEY` or something.

What do you think?

Reply:
good idea